scripts/basic-workflow-tests.sh

```diff
@@ -110,12 +110,16 @@ test_taxonomy() {
 
     test -d taxonomy || git clone https://github.com/instructlab/taxonomy || true
 
-    step Make new taxonomy
-    mkdir -p taxonomy/knowledge/sports/overview/softball
-
-    step Put new qna file into place
-    cp "$SCRIPTDIR"/test-data/basic-workflow-fixture-qna.yaml taxonomy/knowledge/sports/overview/softball/qna.yaml
-    head taxonomy/knowledge/sports/overview/softball/qna.yaml | grep --color '1st base'
+    step Update taxonomy with sample qna additions
+    mkdir -p taxonomy/knowledge/sports/overview/e2e-softball
+    mkdir -p taxonomy/compositional_skills/extraction/answerability/e2e-yes_or_no
+    mkdir -p taxonomy/compositional_skills/extraction/inference/qualitative/e2e-siblings
+
+    step Put new qna files into place
+    cp "$SCRIPTDIR"/test-data/e2e-qna-knowledge.yaml taxonomy/knowledge/sports/overview/e2e-softball/qna.yaml
+    head taxonomy/knowledge/sports/overview/e2e-softball/qna.yaml | grep --color '1st base'
```

Comment:
Why `head`? Could we get multiple `'1st base'` strings in the file? Also I think the error handling for this case could improve. How about we put this through a `if ! command` and log an error message?

Reply:
This is not new code

Reply:
Oh.. right. Nevermind then.